python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -791,26 +791,6 @@ def test_empty_pd_series(basic_store):
     assert basic_store.read(sym).data.empty
 
 
-def test_empty_pd_series_type_preservation(basic_store):
```

Comment:
ArcticDB should not preserve the type anymore. Empty columns will be assigned the (internal) empty type, which translates to Python's object type.